waspc/data/Generator/templates/server/src/jobs/_job.ts

```diff
@@ -0,0 +1,21 @@
+{{={= =}=}}
+import prisma from '../dbClient.js'
+import type { JSONValue } from '../_types/serialization.js'
+import { createJob, type JobFn } from './{= jobExecutorRelativePath =}'
```

Comment:
Don't your new import builders in Haskell cover this case too? I might be mixing stuff up.

Reply:
You are right, they do, but we have a complex import statement here which is not supported by the new import builder 😢

Reply:
Right, the type part... Should we support it, leave a todo or something?